packages/fuels-abigen-macro/src/lib.rs

```diff
@@ -90,10 +102,28 @@ impl ParseInner for ContractArgs {
             let literal = input.parse::<LitStr>()?;
             (literal.span(), literal.value())
         };
+
+        if !input.is_empty() {
+            input.parse::<Token![,]>()?;
+        }
+
+        let binary_filepath = if !input.is_empty() {
```

Comment:
it should probably be refactored with the statement below IMO

Reply:
What would be the better way to do it or what is the problem with the way it is now?

Reply:
I would do a full branch with `!input.is_empty`, no? Since you're testing that condition twice?